git/repo/base.py

```diff
@@ -296,6 +296,16 @@ def __init__(
         else:
             self.odb = odbt(rootpath)
 
+        # Check dubious ownership
+        try:
+            if not (self.bare or Git.is_cygwin()):
+                _ = self.git.status()
+        except GitCommandError as err:
+            if "detected dubious ownership" in err.stderr:
+                err_msg = f"Detected dubious ownership in repository at: {epath}"
```

Comment:
I feel that for many, this message will still not help them understand what's going on, despite it being official `git` lingo.

It think what would help is if it would mention a remedy, like `If this is unintended, run "git config --global --add safe.directory {epath}"`.

Reply:
ok, I tried to be minimalist but agree with you the remedy instruction could be helpful